### DISCUSSION TITLE
IP68 Gehäuse mit ext Antenne

### DISCUSSION THREAD

**Discussion by dersch81** [tbnobody/OpenDTU#247] [Show and tell]
Hallo liebe OpenDTU'ler,

ich möchte euch gerne meine Lösung für ein Outdoor Gehäuse vorstellen. Ich habe keinen 3D Drucker und daher brauchte ich eine andere Lösung die auch mal Regen aushält und nach "Keep it simple aber gescheit" ausgelegt ist. 

![20221010_114629](https://user-images.githubusercontent.com/13754891/196877463-9b82446a-6ece-437f-a874-1ddef402f10c.jpg)

![20221020_083331](https://user-images.githubusercontent.com/13754891/196877492-e04d2cf1-e9d3-4ef4-bdb3-c07cde1d3a28.jpg)

Das Gehäuse findet ihr hier: https://www.amazon.de/gp/product/B08QV1FNV4

**Comment by ghost** [comment]
Sehr schön und wirklich einfach umsetzbar. Auch die Durchführung für die externe Antenne ist kreativ gelöst.

**Reply by dersch81** [reply]
Danke :) Und die Antenne ist dicht. Das habe ich auch schon öfters mit DIY Wlan Routern gemacht die mitten im Wetter hängen.

**Comment by KlausStangl** [comment]
Sehr gute Lösung, Danke fürs Teilen!

**Comment by geraldriess** [comment]
Gute Lösung! Danke dir!

**Comment by stefan123t** [comment]
Habe eine WISKA 80er Dose mit IP68 aus dem Baumarkt für 3,59 verwendet. Die hat den typischen Bender Charme mit seinem Thingy. Hier ein Bild mit nem LoRaWAN Modul, daher nur eine Antenne / Thingy.

![image](https://user-images.githubusercontent.com/11159753/202819227-fceb89dd-09c5-4d2f-b02b-d7c352a35368.jpeg)

**Reply by dersch81** [reply]
Auch ne nette Lösung :)

**Comment by joschua234** [comment]
Sehr schöne Idee :) 
Wie habt ihr das mit der Stromversorgung gelöst? 5V Netzteil im Trockenen an einer Steckdose und per langem USB-Kabel verbunden?

**Reply by dersch81** [reply]
Ich habe ein Netzteil in einem geschüzten Stromverteiler um die Ecke und die 5v mit 2x 0,75mm² direkt in das Gehäuse geführt.
